src/components/StyleGuide/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import Heading from "../Heading/index";
+import styled from "styled-components";
+import Button from "../Button/index";
+
+const Wrapper = styled.div`
+  font-family: ${props => props.theme.font.family.base};
+`;
+
+class StyleGuide extends Component {
+  render() {
+    return (
+      <Wrapper>
+        <Heading>Akai Jobs Style Guide</Heading>
+        <Button primary>Primary button</Button>
+        <Button>Secondary button</Button>
+      </Wrapper>
+    );
+  }
+}
+
+
+export default StyleGuide;
```

Comment:
We could use React Storybook istead of createing StyleGuide component
https://github.com/akai-org/jobs-web/issues/13

Reply:
Ok, I'll remove it for now then.